micro-ui/web/micro-ui-internals/packages/svg-components/CHANGELOG.md

```
@@ -1,6 +1,9 @@
 # Changelog
 
-## [1.0.15]  [30-April-2025]
+## [1.0.18]  [13-May-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)